helix-core/src/main/java/org/apache/helix/manager/zk/client/HelixZkClient.java

```diff
@@ -97,20 +97,34 @@ default void unsubscribeStateChanges(IZkStateListener listener) {
 
   void createEphemeral(final String path);
 
+  void createEphemeral(final String path, final String sessionId);
+
   void createEphemeral(final String path, final List<ACL> acl);
 
+  void createEphemeral(final String path, final List<ACL> acl, final String sessionId);
+
   String create(final String path, Object data, final CreateMode mode);
 
-  String create(final String path, Object datat, final List<ACL> acl, final CreateMode mode);
```

Comment:
nit, not need to make this correction. This make confuse people. Either way, up to you.